subtests/docker_cli/run_cgroups/run_cgroups.py

```diff
@@ -55,3 +56,42 @@ class memory_no_cgroup(memory_base):
 
 class memory_negative(memory_base):
     pass
+
+
+class cgroup_parent_invalid_name(cgroup_parent_base):
+    """
+    Invalid value for --cgroup-parent option
+    """
+
+    def initialize(self):
+        super(cgroup_parent_invalid_name, self).initialize()
+        self._setup("/foobar",
+                    "",
+                    "Error response from daemon: Cannot start container {cid}:"
+                    " [8] System error: Invalid slice name /foobar")
```

Comment:
I don't believe I wrote this test, or at least haven't looked at it in a few decades or three.  The code is normally all in the parent class, so the empty sub-subtest class is really just there to create the name.  The name needs to be held so it can be found/loaded by the ``SubsubtestCaller()`` (or w/e it's name is) in the subtest API.  No fancy IDE's just two side-by-side terminal windows :grinning: